src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -85,33 +103,55 @@ export const ChannelView: React.FC = () => {
     throw videosError
   } else if (error) {
     throw error
+  } else if (errorSearch) {
+    throw errorSearch
   }
 
-  const handleSorting = (value?: VideoOrderByInput | null | undefined) => {
+  const handleSetCurrentTab = async (tab: number) => {
+    setIsSearching(false)
+    setCurrentVideosTab(tab)
+  }
+  const handleSorting = (value?: VideoOrderByInput | null) => {
     if (value) {
       setSortVideosBy(value)
       refetch({ orderBy: value })
     }
   }
   const handleOnResizeGrid = (sizes: number[]) => setVideosPerRow(sizes.length)
   const handleChangePage = (page: number) => {
-    setCurrentPage(page)
+    isSearching ? setCurrentSearchPage(page) : setCurrentPage(page)
   }
-  const handleSetCurrentTab = async (tab: number) => {
-    setCurrentVideosTab(tab)
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Maybe we could wrap this input in form, I think we will not need all these keyboard events If Im not mistaken